src/cheri_insts.sail

```diff
@@ -778,6 +865,20 @@ function clause execute(CRAM(rd, rs1)) = {
 }
 
 union clause ast = CTestSubset : (regidx, regidx, regidx)
+/*!
+ * Integer register *rd* is set to 1 if the **tag** fields of capability
+ * registers *cs1* and *cs2* are the same and the bounds and permissions of
+ * *cs2* are a subset of those of *cs1*.
+ *
+ * ## Notes
+ *
+ * - The operand order for this instruction is reversed compared with the
+ *   normal RISC-V comparison instructions, but this may be changed in future.
+ *
+ * - The **otype** field is ignored for this instruction, but an alternative
```

Comment:
That is rather surprising behaviour. I guess it is okay for sentries, but not necessarily for user defined otypes.

Reply:
Yeah, something I noticed whilst writing this so this note is an original creation rather than a regurgitation of something MIPS has.